src/main/java/de/doubleslash/keeptime/common/SvgNodeProvider.java

```diff
@@ -0,0 +1,50 @@
+package de.doubleslash.keeptime.common;
+
+import javafx.geometry.Bounds;
+import javafx.scene.Node;
+import javafx.scene.shape.SVGPath;
+import org.w3c.dom.Document;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class SvgNodeProvider {
+
+    private static String getSvgPathWithXMl(Resources.RESOURCE resource) {
+        String svgPath;
+        Document document;
+        DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+        DocumentBuilder db = null;
+
+
+        try {
+            db = dbf.newDocumentBuilder();
+        } catch (ParserConfigurationException e) {
+            throw new RuntimeException(e);
```

Comment:
can you add a string message to the exception you throw, indicating what was tried to be done? See constructor https://docs.oracle.com/javase/7/docs/api/java/lang/RuntimeException.html#RuntimeException(java.lang.String,%20java.lang.Throwable)